src/System.Web.WebPages/Helpers/AntiXsrf/AntiForgeryWorker.cs

```diff
@@ -104,7 +104,11 @@ public TagBuilder GetFormInputElement(HttpContextBase httpContext)
                 // Adding X-Frame-Options header to prevent ClickJacking. See
                 // http://tools.ietf.org/html/draft-ietf-websec-x-frame-options-10
                 // for more information.
-                httpContext.Response.AddHeader("X-Frame-Options", "SAMEORIGIN");
+                const string xFrameHeaderName = "X-Frame-Options";
```

Comment:
Name will still fail a build with the SA130 error. (Might be worth setting up AppVeyor for this repo.)

Reply:
@dougbu I uploaded the `string const` change to my repo, Do I need to initiate another pull request or anything?

Do you need further changes to get past the SA130 error?

Reply:
Yes, please renamed the `const` to something like `frameHeader` or `frameHeaderName`. That'll get command-line builds working.

Reply:
All set, let me know if you need anything else.  Thanks.